build/ublue-os-just/ublue-os-just.spec

```diff
@@ -1,7 +1,7 @@
 Name:           ublue-os-just
 Packager:       ublue-os
 Vendor:         ublue-os
-Version:        0.20
+Version:        0.22
```

Comment:
Small thing but, I'm curious why this jumped from 0.20 to 0.22? Where is 0.21?

Reply:
Oh. I answered myself. Looks like we had a 0.21 changelog entry which missed the package version update.

Reply:
yep so i fixed it when i noticed it